sdk/core/Azure.Core/tests/OperationPollerTests.cs

```diff
@@ -49,7 +49,7 @@ public void WaitForCompletionResponseAsyncCancelled()
         {
             var cts = new CancellationTokenSource();
             var poller = new OperationPoller(new TestDelayStrategy(TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(1), cts));
-            var operation = new OperationInternal(new ClientDiagnostics(ClientOptions.Default), new EndlessOperation(), new MockResponse(200));
+            var operation = new OperationInternal<int>(new ClientDiagnostics(ClientOptions.Default), new EndlessOperation(), new MockResponse(200));
```

Comment:
Is it possible to provide test coverage for OperationInternal still?

Reply:
This is a test for `OperationPoller`. There is no signature that accepts `OperationInternal` anymore.